src/Game.java

```diff
@@ -43,25 +28,36 @@ public void printCharArray() {
     }
 
 
+    public void handleCommand(String input, Hero hero, Stone stone) {
+
+        //check for special words (maybe should store all special words in some place?)
+        if(input.toLowerCase().equals("quit") ||
+                input.toLowerCase().equals("reset")) {
+            return;
+        }
 
-    public void handleCommand(String input) {
         char command = input.charAt(0);
-        Coordinates futureMove = hero.checkRoad(command);
+        if ( command != 'w' && command != 's' && command != 'a' && command != 'd'  ){
+            System.out.println("Command not recognized.");
+            System.out.println("Use W, S, A, D to move around.");
+            System.out.println("Type QUIT to quit the game or RESET to reset the level.");
```

Comment:
Zrób sobie prosty wrapper na system.out i input, żeby to też móc przetestować testując Game.

Ogólnie jak w komentarzu poniżej to będą następne testy, gdzie assert będzie sprawdzał czy dana metoda mocka się zawołała